clients/php/predis.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "predis",
+    "description": "A flexible and feature-complete Redis client for PHP.",
+    "twitter": [
+        "JoL1hAHN"
```

I'm the new owner for the last couple of years.

```suggestion
        "tillkruss"
```

---

So this field is actually a leftover from the old structure of these json files, and so it won't be rendered anywhere when this goes to the website. I removed this for all of the other JSON client files as well and left it here by mistake, so I think It's better to just remove this field altogether :)

---

I agree. Let's let the github be the main access page, and then individuals can include links like twitter there.